### ISSUE TITLE
Text elements containing only numbers are always rendered top left

### ISSUE THREAD

**Issue by xetra11** [KlausSchaefers/figma-low-code#10]
When I create a text element and have **only** numbers in it, it will be rendered in the top left corner. As soon as there is a character in the text it is rendered as expected.

**String/Character**
> Figma
![image](https://user-images.githubusercontent.com/8149023/95128393-a7955e80-0759-11eb-895a-7666cb57dc8d.png)
> Vue.js Rendering
![image](https://user-images.githubusercontent.com/8149023/95128481-c693f080-0759-11eb-9802-0bf5f284c7bd.png)

**Number Only**
> Figma
![image](https://user-images.githubusercontent.com/8149023/95128535-dd3a4780-0759-11eb-9a31-21df4970537a.png)
> Vue.js Rendering
![image](https://user-images.githubusercontent.com/8149023/95128608-f8a55280-0759-11eb-979c-6f57cc57320e.png)

Code:
```html
<template>
  <Figma :figma="figmaConfig"></Figma>
</template>

<script>
export default {
  name: 'FigmaLowCodeTest',
  data: function () {
    return {
      figmaConfig: {
        figmaFile: '<fileId>',
        figmaAccessKey: '<key>'
      }
    }
  }
}
</script>

```

**Comment by KlausSchaefers** [comment]
Hm, this is a funny one. Ca you share the Figma file with me?

**Comment by Fjgcornejo** [comment]
That was happening to me also, I realized that the name of the figma elements (on the left pane) is used for CSS and class names, and if the element is called for example, "123" it won't have any styles or HTML classes on it.
If this is your case, try changing it to something like "dummyText".

**Comment by KlausSchaefers** [comment]
Ah, ok. Thanks for the investigation! I will fix that in the next release!

**Comment by KlausSchaefers** [comment]
Should be fixed with vue-low-code 0.4.31.

**Comment by xetra11** [comment]
works thx!
